xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -438,3 +438,14 @@ bool CAndroidUtils::SupportsMediaCodecMimeType(const std::string& mimeType)
 
   return false;
 }
+
+std::pair<bool, bool> CAndroidUtils::GetDolbyVisionCapabilities()
+{
+  bool displaySupportsDovi = CAndroidUtils::GetDisplayHDRCapabilities().SupportsDolbyVision();
+  bool mediaCodecSupportsDovi = CAndroidUtils::SupportsMediaCodecMimeType("video/dolby-vision");
```

Comment:
```suggestion
  const bool mediaCodecSupportsDovi = SupportsMediaCodecMimeType("video/dolby-vision");
```
Class name prefix is not needed since we are in the same class
Same in `displaySupportsDovi`